### ISSUE TITLE
Loras in wildcard prompts

### ISSUE THREAD

**Issue by cobaltbeau** [AUTOMATIC1111/stable-diffusion-webui-wildcards#29]
I don't think it's correctly handling loras in wildcard prompts in a batch 

I have a wildcard file of the following lines:

<lora:TLOK_Asami_Sato_v4:0.6>, TLOK_Asami_Sato, \green eyes\, black hair, main_outfit
<lora:YangXiaoLong:1>, ((yellow hair)), yang xiao long, (prosthetic arm), smile, ahoge purple eyes, brown jacket, waist coat
<lora:nora_valkyrie:0.6>, nora_valkyrie, looking at viewer, smile, short hair, [[blue eyes]], gloves, jacket, pink skirt, pink belt, short sleeves, puffy sleeves, fingerless gloves, orange hair

They are character loras for Asami Sato from Legend of Korra, and Yang Xiao Long + Nora Valkyrie from RWBY, for visual reference.  When I run this prompt using a batch size of 1 and batch count of 6, I get what I'd expect:

![image](https://github.com/AUTOMATIC1111/stable-diffusion-webui-wildcards/assets/5383283/a4fbeb70-e312-4dfd-87fe-3167207ea517)

But when I generate with a batch size of 6 and batch count of 1, I think it's including the lora used for the first image in all the images generated:
![grid-0007](https://github.com/AUTOMATIC1111/stable-diffusion-webui-wildcards/assets/5383283/c4e5413b-1ab1-46c3-a94e-971e3c24e963)
![grid-0008](https://github.com/AUTOMATIC1111/stable-diffusion-webui-wildcards/assets/5383283/926975a3-0df0-4c39-92dd-a1d9b1649a0f)
![grid-0009](https://github.com/AUTOMATIC1111/stable-diffusion-webui-wildcards/assets/5383283/43b26709-052d-4ba0-a52e-f41c7010e048)

**Comment by WesleyNery** [comment]
Yeah I see this problem as well. It happens especially when using highres.fix for me. It picks one of the options based on the seed however it seems that nothing is passed on to the highresfix prompt so the image changes or blends
